polyfill/lib/calendar.mjs

```diff
@@ -117,7 +117,11 @@ export class Calendar {
     return GetSlot(this, CALENDAR_ID);
   }
   static from(item) {
-    if (ES.Type(item) === 'Object') return item;
+    if (ES.Type(item) === 'Object') {
+      if (!('calendar' in item)) return item;
+      item = item.calendar;
+      if (ES.Type(item) === 'Object' && !('calendar' in item)) return item;
```

Comment:
What's the use case for a calendar property that's not a calendar object yet?

Reply:
The calendar could be given as a string in the property bag:
```js
const myPropertyBag = { year: 2020, month: 10, day: 20, calendar: 'gregory' };
Temporal.Calendar.from(myPropertyBag);
```